### PR TITLE
Add filename

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: test/integration/local/process-acknowledgement.test.js [hunk 1]
```diff
@@ -83,6 +83,11 @@ describe('process acknowledgement', () => {
     expect(mockSendBatchMessages.mock.calls[0][0][2].body.message).toBe('Journal JN12345678 has been created Validation failed Line : 21.')
   })
 
+  test('sends filename if file valid', async () => {
+    await processing.start()
+    expect(mockSendBatchMessages.mock.calls[0][0][0].body.filename).toBe(VALID_FILENAME)
+  })
+
```
Comment (Kirsten1822) [inline test/integration/local/process-acknowledgement.test.js:90]: I was thinking this test should maybe be above the 'sends success if file valid' test

File: test/integration/local/process-acknowledgement.test.js
Comment (Kirsten1822) [inline test/integration/local/process-acknowledgement.test.js:1]: I think the lets should be defined below the TEST_FILE, TEST_INVALID_FILE, VALID_FILENAME, INVALID_FILENAME consts

Comment (johnwatson484) [reply test/integration/local/process-acknowledgement.test.js:1]: Yeah I agree, but don't really want to rewrite all of the test files in the scope of this small ticket.

Maybe we can do that as part of a bigger tidy up ticket when we have more capacity?

File: test/integration/local/process-payment-file.test.js
Comment (Kirsten1822) [inline test/integration/local/process-payment-file.test.js:1]: I think the consts and lets at the top of the file should be separated a bit more

Comment (Kirsten1822) [reply test/integration/local/process-payment-file.test.js:1]: Also there's a random mock below the lets

Comment (Kirsten1822) [reply test/integration/local/process-payment-file.test.js:1]: Unless it has to be there?

Comment (Kirsten1822) [inline test/integration/local/process-payment-file.test.js:1]: Does the '// helper function to upload files to blob storage' comment need to be there?

File: test/integration/local/process-payment-file.test.js [hunk 2]
```diff
@@ -47,11 +49,6 @@ describe('process payment files', () => {
     await batchFileUploader(mockReturnFileNames, RETURN_TEST_FILE)
     await processing.start()
 
-    const inboundFileList = []
-    for await (const item of container.listBlobsFlat({ prefix: config.storageConfig.inboundFolder })) {
-      inboundFileList.push(item.name)
-    }
-
```
Comment (Kirsten1822) [inline test/integration/local/process-payment-file.test.js:54]: Why were these removed?

File: test/integration/local/process-payment-file.test.js
Comment (Kirsten1822) [inline test/integration/local/process-payment-file.test.js:1]: There's still one more spelling error of acknowledgement in this file: line 129

File: test/integration/local/process-return.test.js
Comment (Kirsten1822) [inline test/integration/local/process-return.test.js:1]: I think the two constants RESPONSE_REJECTED and SOURCE should be defined above TEST_FILE and TEST_INVALID_FILE

Comment (Kirsten1822) [inline test/integration/local/process-return.test.js:1]: Space needed between the afterEach and the describe

File: test/integration/local/process-acknowledgement.test.js
Comment (Kirsten1822) [reply test/integration/local/process-acknowledgement.test.js:1]: It's up to you if that's what you want to do :)

Comment (johnwatson484) [reply test/integration/local/process-acknowledgement.test.js:1]: I was thinking more generally for comments like this as although this is only moving two lines, doing that for each comment adds more risk to the change as more and more areas are touched.

But the tests in this repo are decent and I think it's small enough not to add too much time and risk to the change.

Decision made, I'll update the minor quick wins but I'll still create a ticket for a more considered tidy up.

File: test/integration/local/process-payment-file.test.js
Comment (johnwatson484) [reply test/integration/local/process-payment-file.test.js:1]: Nah, deleted

Comment (johnwatson484) [reply test/integration/local/process-payment-file.test.js:1]: Fixed

Comment (johnwatson484) [reply test/integration/local/process-payment-file.test.js:1]: It does not.  Moved

File: test/integration/local/process-payment-file.test.js (hunk 2 above)
Comment (johnwatson484) [reply test/integration/local/process-payment-file.test.js:54]: They get all of the files that are in the inbound folder.  But the assertions don't check what's in that folder so why bother getting that info.

Suspect originally the checking of what's in archive and inbound was done in one test, but were split at some point but removing the redundant checking steps was missed.

File: test/integration/local/process-acknowledgement.test.js (hunk 1 above)
Comment (Kirsten1822) [reply test/integration/local/process-acknowledgement.test.js:90]: I think this comment may have been missed?

Comment (johnwatson484) [reply test/integration/local/process-acknowledgement.test.js:90]: Ah sorry, I did move it but then moved it back later.  The tests are in order of the properties in the message so thought best to keep where it is for consistency.
